docs/docsite/rst/plugins/lookup.rst

```diff
@@ -12,6 +12,7 @@ Lookup plugins are an Ansible-specific extension to the Jinja2 templating langua
 .. note::
    - Lookups are executed with a working directory relative to the role or play,
      as opposed to local tasks, which are executed relative the executed script.
+     The ``file`` lookup plugin is exception to this as it has a list of path candidates in which it searches.
```

Comment:
Those that look for files, vars or templates with a relative path will use the ``ansible_search_path``, to find those files.

Reply:
probably with a link to https://docs.ansible.com/ansible/latest/user_guide/playbook_pathing.html